notebook/static/notebook/less/highlight.less

```diff
@@ -64,6 +64,12 @@ Adapted from GitHub theme
   color: #AA22FF;
 }
 
+//fix issue #4206, overwrite .highlight-comment class
+.highlight-comment{
+  color:#007979;
```

Comment:
Do we need to add a new rule? Can we not modify the existing one at line 32?

Reply:
Sorry for the mistakes made, I had updated the screenshots and code now.

Reply:
> Do we need to add a new rule? Can we not modify the existing one at line 32?

Sorry for the mistakes made, I had updated the screenshots and code now.